### DISCUSSION TITLE
Use Power to and Power from grid in Energy Dashboard

### DISCUSSION THREAD

**Discussion by jonassmets** [joleys/niko-home-control-II#194] [Q&A] [answered]
Is there a way to use the Niko Home Control entities in the Home Assistant energy dashboard? 
I have 2 entities: 
- energyhome Electrical Power from Grid
- energyhome Electrical Power to Grid
In Niko Home Control they provide live energy information. But I'm not able to use them in HAS because they don't have the device_class energy assigned? 

I'm currently using EnergyID connected to my provider. But this updates the dashboard only once every 24 hours. Alternatively, I could go for a p1 splitter and a p1 connector from Homewizard? 

Thanks
Jonas

**Comment by tijsverkoyen** [comment, marked as answer]
I think https://github.com/joleys/niko-home-control-II?tab=readme-ov-file#calculated-totals and https://github.com/joleys/niko-home-control-II/discussions/187 are relevant?
